
#AGV底盘信息
Wheel_Diameter : 0.195
Wheel_Track: 0.405
Reduction_Ratio: 40
Encoder_Accuracy: 10000
Velocity_Factor: 240

#串口参数设置
serial_dev: /dev/usb-ttl
Baud_Rat: 115200

#超声波距离判断
UNDM_Dis: 40
UNDM_Correct: true
